### ISSUE TITLE
Make logger customizable (again)

### ISSUE THREAD

**Issue by alexanderadam** [mbreit/pg_jobs#28]
If you really insist on the Rails logger use at least something like

```ruby
module PgJobs
  mattr_reader :logger
  self.logger
    @logger ||= Rails.logger # or maybe even rather ActiveJob::Base.logger
  end
  …
end
```

Because then people can still override it if they need.

Besides from that: thank you for making `pg_jobs`! :+1:

**Comment by mbreit** [comment]
@alexanderadam Thank you for your suggestion and your kind words. And sorry this took so long.
I made a PR #29 for this and changed the default logger to `ActiveJob::Base.logger` as you suggested. I am not really sure what be best default would be (most of the other ActiveJob queues create their own logger by default), but the ActiveJob logger should work for the most small Rails projects I guess.

Could you take a look at the PR #29 to see if that would work for you? Then I will include that in the 0.2.0 release.

**Comment by alexanderadam** [comment]
Hi Moritz,

awesome, it seems to work great (if I didn't miss anything). :+1:

**Comment by mbreit** [comment]
Thanks for your feedback. I have merged the MR so it will be part of 0.2.
